modules/core/quick/tests/src/Functional/QuickFormTest.php

```diff
@@ -146,6 +146,9 @@ public function testQuickForms() {
     ]);
     $config_entity->save();
 
+    // Rebuild routes.
+    \Drupal::service('router.builder')->rebuildIfNeeded();
+
```

Comment:
Would be nice if this line wasn't required in tests, but makes sense it may be. Seems like the next request in this test should trigger a rebuild of the routes but maybe that isn't the case? Just want to make sure we tried

Reply:
Yea that line is required for the tests to pass. I just retested locally with and without to confirm.